openapi/components/schemas/LtftApplicationAdminSummary.yml

```diff
@@ -0,0 +1,41 @@
+type: object
+readOnly: true
+properties:
+  id:
+    type: string
+    format: uuid
+    example: fc13458c-5b0b-442f-8907-6f9af8fc0ffb
+  personalDetails:
+    allOf:
+      - $ref: ./PersonalDetailsName.yml
+      - $ref: ./PersonalDetailsRegistration.yml
+  programmeName:
+    type: string
+    example: General Practice
+  proposedStartDate:
+    type: string
+    format: date
+  submissionDate:
+    type: string
+    format: date
+  daysToStart:
+    type: number
+  shortNotice:
+    type: boolean
+  tpd:
+    type: object
+    required:
+      - tpdEmail
+      - tpdEmailStatus
+    properties:
+      email:
+        type: string
+        format: email
+      emailStatus:
+        type: string
+        enum: [SENT, BOUNCED, REJECTED]
+  status:
+    $ref: ../properties/FormStatus.yml
+  assignedAdmin:
+    type: string
```

Comment:
Is this going to be the username of the TIS user who is assigned (i.e. HeeUser.name, or a composite of HeeUser.fistname and lastname) or freetext?

Reply:
Not sure what the desire is, we've got no tickets that far ahead and no way (currently) to assign that admin anyway,